js/home.js

```diff
@@ -128,6 +115,119 @@ function sortPokemonList (sorting, refresh) {
   addFavoriteButtonEvent()
 }
 
+function format ( d ) {
+    // `d` is the original data object for the row
+     return '<table class="table table-condensed table-hover" id="'+d.pokemon_id+'" style="width:100%;">'
+        + '<thead>'
+          + '<tr>'
+            + '<th width="5%"><input type="checkbox" id="checkall"></th>'
+            + '<th>'
+              + '<span class="glyphicon glyphicon-star favorite-yellow"></span>'
+            + '</th>'
+            + '<th>Name</th>'
+            + '<th>Nickname</th>'
+            + '<th>CP</th>'
+            + '<th>IV (A/D/S)</th>'
+          + '</tr>'
+        + '</thead>'
+        + '</table>';
+}
+
+function dataTables(pokemon) {
+    var table = $('#pokemon-data').DataTable( {
+      data: pokemon,
+      className: 'details-control',
+      bPaginate: false,
+      bInfo: false,
+      columns: [
+      {
+        className:      'details-control',
+        orderable:      false,
+        data:           null,
+        defaultContent: ''
+
+      },
+      { data: "pokemon_id" },
+      { data: "name" },
+      { data: "count" },
+      { data: "candy" },
+      { data: "evolves" }
+      ],
+      order: [[1, 'asc']]
+    } );
+
+        // Add event listener for opening and closing details
+        $('#pokemon-data tbody').on('click', 'td.details-control', function () {
+          var tr = $(this).closest('tr');
+          var row = table.row( tr );
+
+          if ( row.child.isShown() ) {
+            // This row is already open - close it
+            row.child.hide();
+            tr.removeClass('shown');
+          }
+          else {
+            // Open this row
+            row.child( format(row.data()), 'child').show();
+            tr.addClass('shown');
+            var prepped = prep_display(row.data())
+            sub_datatable(row.data(), prepped)
+          }
+        } );
+
+}
+
+function prep_display(d) {
+
+  for (var i = 0; i < d.pokemon.length; i++) {
+    var poke = d.pokemon[i]
+    var checkBox = '<input type="checkbox" value="' + poke.id.toString() + '"'
+    var favorite = 'glyphicon glyphicon-star-empty'
+    var pokeiv = poke['iv'] + '% (' + poke['attack'] + '/' + poke['defense'] + '/' + poke['stamina'] + ')'
+    var favoriteBool = poke['favorite'] ? 'true' : 'false'
+
+    if (poke.deployed) checkBox += ' disabled'
+    if (poke.favorite) favorite = 'glyphicon glyphicon-star favorite-yellow'
+
+      poke.checkbox = checkBox + '>'
+      poke.favorited = '<span class="favorite ' + favorite + '" id="favoriteBtn" data-pokemon-id="' + poke.id + '" data-pokemon-favorited="' + favoriteBool + '" />'
+      poke.nickname = '<a class="nickname" data-pokemon-id="' + poke.id + '">' + poke.nickname + '</a>'
```

Comment:
If you change this var to `nicknamed`, and then update line 212, I think this is good to merge 👍